variables.tf

```diff
@@ -904,3 +904,22 @@ variable "custom_container_apps" {
   }))
   default = {}
 }
+
+variable "container_app_identities" {
+  description = "Identities to assign to container app"
+  type = object({
+    type : optional(string)
+    identity_ids : optional(list(string))
+  })
+  default = {}
+
+  validation {
+    condition     = anytrue([var.container_app_identities.type == null && var.container_app_identities.identity_ids == null, contains(["SystemAssigned", "UserAssigned", "SystemAssigned, UserAssigned", ], coalesce(var.container_app_identities.type, " "))])
+    error_message = "Identity type must be one of the following: 'SystemAssigned', 'UserAssigned', or 'SystemAssigned, UserAssigned'!"
+  }
```

Comment:
We should leave the validation to the Azure api
If an incorrect value is set, Azure will do the right thing and throw the error
This then mitigates the possibility that Azure will change the allowed values (in which case we would need to update this validation)